3-building-blocks-of-oop-part-2/hometask/src/Bow.ts

```diff
@@ -0,0 +1,14 @@
+import {Weapon} from "./Weapon";
+
+export class Bow extends Weapon {
+    constructor(baseDamage: number, baseDurability: number, value: number, weight: number) {
+        super('bow', baseDamage, baseDurability, value, weight);
+    }
+
+    polish(): void {
+        const effectiveDurability = this.getEffectiveDurability();
+        if (effectiveDurability < 1) {
```

Comment:
```polish(): Increases the instance's durabilityModifier by adding Weapon.MODIFIER_CHANGE_RATE if effective durability after this is less or equal to 1```

Reply:
If I will set <= condition, then we'll have durability 105%, which cannot be. Maybe there is an error in the condition?